internal/health_checks/health_check_error.go

```diff
@@ -0,0 +1,120 @@
+// Copyright 2022 Google LLC
```

Comment:
General comment for this file: The `Action`s have a lot of potential to derive user value in being able to self-troubleshoot, so it's worthwhile to ensure that each `Action` contains a comprehensive, concrete list of steps to perform, or a link to those steps. But many of them are vague, like `"Verify that the port is available"` or `"Check your internet connection"`, and all of the links go to the generic troubleshooting page. This isn't as "actionable" for the user as it could be IMO.

Is there a plan to add more specific steps to either these messages or the troubleshooting page?

Reply:
Yes, I agree. There's more work needed in the `Action` and `ResourceLink` to derive clear value for the end user. There's a planned documentation page release in the health checks project that will list all the ERROR_CODES a source of truth ( CC @jinghan-ma , @mtabasko ). Will work on getting this information in a final version (or close to final for the next iteration of this review).